### DISCUSSION TITLE
Create Initial Liquidity for CRAB Listing on an Alternative Chain and DEX

### DISCUSSION THREAD

**Discussion by sonoferin2000** [ringecosystem/discussion#16] [General]
This proposal urges the DAO to create initial liquidity for CRAB on an alternative chain and list it on a DEX with exposure to price feeds,
 such as Solarbeam on Moonriver. This will help bring visibility to CRAB, increase its market presence, hopefully be able to beifit from price appreciation. THis is something that should have been done much sooner. It has been proposed, but sadly no action was ever taken. 

Motivation
Currently, CRAB only trades on Snowswap, which does not broadcast transactions to major price aggregators like CoinGecko and CoinMarketCap. As a result, CRAB remains invisible to the broader crypto space, limiting its growth potential.

With the bear market behind us, now is the perfect time to take action. We have the opportunity to drive the token price upward and 
 attract new users to the CRAB network by listing it on a DEX with price feed exposure. This should have happened much sooner.

Proposal
   Liquidity Creation
        The DAO should allocate initial liquidity for CRAB on an alternative chain, such as Moonriver.
        The liquidity should be listed on a DEX with price feed exposure, such as Solarbeam DEX.
        This will allow CRAB to be visible on major price tracking platforms like CoinGecko and CoinMarketCap.

Incentivization
   Consider offering incentives for liquidity providers. This will encourage liquidity provision and increase CRAB's liquidity pool.
we have lost a lot of time being isolated like we have.encentivising the pool would give it added attention. We may get more people
 taking part in staking, and trading on snowswap for other assets that they wouldn't have considered before.  

Increased Exposure
  Listing CRAB on a platform with price feed exposure will significantly increase its visibility in the broader crypto space.
  This will make it easier for potential buyers to track CRAB’s price and trade it on accessible platforms. We need more users.

Long-Term Strategy
   Creating liquidity on a DEX will attract new users and increase trading volume.
   This will lay the foundation for long-term growth and adoption of CRAB within the wider ecosystem.

Impact
      Increased Visibility: Listing on a DEX with exposure to price feed aggregators will enhance CRAB’s presence on CoinGecko,
 CoinMarketCap, and other aggregators. Attraction of New Users and Liquidity Providers: Incentives for liquidity provision will 
encourage active participation and increase liquidity. More liquidity and visibility will likely drive up CRAB’s price and market 
recognition. it has been relegated to obscurity only trading on snowswap without price feed exposure. RingDAO should take 
the initiative on this and not rely fully on users to do this. The sooner action is taken, the sooner we can take part in this recent 
period of price appreciation everyone else has experienced and benefited from.

Conclusion
To bring CRAB into the broader crypto space, the DAO must take action to list the token on a DEX with price feed exposure. By doing
 so, we can significantly increase CRAB’s visibility, attract new users, and position it for long-term growth. I strongly urge the DAO to 
implement this proposal and take the necessary steps to create initial liquidity for CRAB.

**Comment by SasoLithops** [comment]
Thank you for putting this up @sonoferin2000 

I agree it would be interesting to se Crab trading on other (para)chain like Moonriver. Ring and KTON exist on many chains why shouldn't Crab(possibly CKTON). 
Also good point is that incentives for liquidity providers would roll some eyes on what Crab chain actually is. 

The huge problem I see for now is what can I actually do with Crab token? Atm not much. So Crab should probably have something to offer before it gets put on DEX  and be incentivised as no one would buy it, yet provide liquidity if he can only swap Crab to Ring or CKTON. I so miss Evolution Land where all tokens of Darwinia had extra usage....

This idea however would be especially good now, after spammening where Kusama proved yet again it is far from what was thought before just being Polkadot testing chain but defenitely its own super tech chain. I watched video on Kusamarian with ERIN GRASMICK, she explains it so much better then I could about Kusama future. I recommend watching it.

**Reply by sonoferin2000** [reply]
> The huge problem I see for now is what can I actually do with Crab token? Atm not much. So Crab should probably
 have something to offer before it gets put on DEX and be incentivised as no one would buy it, yet provide liquidity if 
he can only swap Crab to Ring or CKTON.

Waiting to list CRAB until it “has something to offer” doesn’t hold up to scrutiny. RING and CRAB are essentially the 
same token—like a reflection in a mirror. By this logic, RING wouldn’t be tradable either, yet it is. Realistically, what can 
we currently do with RING? Its primary functions are for transaction fees, staking, and trading for another token. While
 these are valuable uses, they don’t make RING inherently exceptional beyond that scope. CRAB serves the same
 purpose, so this reasoning doesn’t justify not listing CRAB on another DEX.

In fact, you previously proposed listing CRAB here: https://github.com/orgs/ktondao/discussions/28. Why the sudden 
reluctance? I firmly believe listing CRAB should be incentivized. To address potential concerns about incentives, I’m 
even willing to personally cover the costs for a month or so. This ensures that incentivization doesn’t become a 
sticking point.

CRAB has been overlooked for so long that it’s almost forgotten. It doesn’t even appear on CoinGecko’s radar when 
people search for low-market-cap opportunities. There’s no compelling reason for this continued lack of visibility—it 
could be listed and gain attention. Incentivizing liquidity now isn’t just about farming rewards; it’s a way to boost 
awareness and put CRAB on people’s radar.

Listing CRAB on an active DEX would naturally require liquidity providers to obtain both CRAB and xWRING 
to participate, given the current lack of other tradable assets aside from CKTON.this opens up an additional market 
for RING So This creates a direct incentive for users to interact with the Crab network and potentially the Darwinia 
network as well. Some of these individuals might even choose to explore staking opportunities on Crab or Darwinia, 
further expanding network activity and participation. There’s really no downside to this approach—it fosters 
engagement, awareness, and utility for the ecosystem.

Frankly, I shouldn’t have to make such a hard case to have CRAB listed. Team should have done this from the very start.
Liquidity providers care about farming opportunities and competitive yields, not necessarily deeper utility.Thats a fact.
 If providing liquidity for CRAB offers solid rewards, they’ll participate. Tokens with less utility—meme coins,have drawn 
significant attention, with no utility really at all,  so dismissing this idea outright seems unfair. Releasing CRAB to a 
broader audience carries significant upside potential with speculators.  Who isn't hunting for potential low cap gems?

That said, CRAB won’t gain traction or attract holders/stakers, if it remains confined to this relatively obscure network only.
 Expanding to another chain, such as Moonriver or any other compatible ones, with an active DEX ecosystem, is a strategic 
and necessary step forward. 

@hackfisher, @2xJDubs, @Laki389

**Reply by SasoLithops** [reply]
Oh that was long ago. As you see nothing was done in that direction, no iniciative was taken by anyone. 

If you read again my reply I am NOT against listing Crab on DEX, I just had this major concern I pointed out.bAtm not sure how much new development is being worked on Crab at all. @hackfisher will know. Other points I mention were pros. 

Crab used to be traded on CEXs I personaly remember being on MEXC as I bought them there, but we got delisted. Also liquidity with USDC with huge incentives was provided in Snowswap for half a year. But died after incentives run out. And Crab was widely traded back then and was used in Evolution land.

Two more things.
I wouldn't compare Crab with memes....people constantly aping into them....especialy on Solana....on Cosmos for example and Polkadot they can't get traction....Otherwise we could have Crab as meme:-)
And who would provide liquidity and rewards?

**Reply by sonoferin2000** [reply]
I felt you were being dismissive to this because it lacks utility. Sorry for any  misunderstanding I had to your comment,I read it in a certain way. No, I'm not comparing CRAB to meme tokens, but I don't think it's fair to say no one would buy it. Listing it could drive the price in a positive direction for a change. I think that simply making it more accessible could do a lot of good for the token and the ecosystem. I think we need more wallets holding CRAB or RING. People care more about a token they actually own, versus ones they don't. Listing the token will allow people to buy and hold it, which can't hurt. Rewards is a good way to lure new faces to the token and project.The goal is to create more holders and not just have the same 50 people locking and dumping tokens. More utility will come, price appreciation could stimulate things. currently the exchange rate is 26 crab per 1 ring, that could easily change by haveing additional buyers on another chain farming liquidity., or simply speculators finally  seeing it on their radar when they search by marketcap, or polkadot/kusama ecosystem etc., I want more eyeballs on our tokens, and project. I feel this will really helps.

**Reply by 2xJDubs** [reply]
I think listing CRAB on Solarbeam, MoonRiver, or getting it to the more active Moonbeam and setting up a pool on Solarflare makes a lot of sense.  

A pool on Solarbeam would be easier for people to figure out how to buy and move back and forth since Crab Network and MoonRiver are already connected and xcCRAB already exists, though it may need renewed mapped with CRABs change to more of an EVM based token.  But I think doing this doesn't help CRAB be seen as MoonRiver activity has decreased immensely from the time it became a parachain and I have concerns that the Moonbeam foundation may deprecate it.

A pool on Solarflare while be less likely to disappear due to a chain deprecation but is more of a challenge for users to navigate getting CRAB there by sending it cross chain from Crab to Darwinia to get WCRAB and then cross chain it to MoonBeam from Darwinia.  If we can simplify this jump using para.link.

**Comment by sonoferin2000** [comment]
I think project itself should initiate liquidity. after all it is their project,they should initially start the pool. community can also contribute once they know it has been established. ill contribute to it,Darwinia can announce it through their channels, letting the diaspora know that it's open, and Solarbeam could do likewise, project can incentivise or not, at least it is out in the wild.  Like I said, I'm willing to incentivize it personally at the start if they don't.  but Im not going to give up my rewards indefinitely, Im only doing so much, but ill do it for a month or so if they don't step up. there are also SOLAR incentives to consider.

**Reply by SasoLithops** [reply]
You shouldn't do this alone. You think in a way it was done for Ring on Hydration. That would make sense. Just not sure they have a canary chain on Kusama.
It shows to me this debate is closed, reopen it please. I will tag some to express their opinion or this.

**Reply by sonoferin2000** [reply]
it shows as being open
